clintrials/core/simulation.py

```
@@ -302,7 +176,7 @@ def __init__(self, design, outcome_generator=None, recruitment_stream=None):
         self.outcome_generator = outcome_generator
         self.recruitment_stream = recruitment_stream
 
-    def run(self, mode="iterative", n_sims=1, cohort_size=1, show_progress=False, **kwargs):
+    def run(self, mode='iterative', n_sims=1, cohort_size=1, show_progress=False, **kwargs):
```

Ruff (0.15.21)

[warning] Boolean default positional argument in function definition

(FBT002)